dataeng/jobs/analytics/SnowflakeCollectMetrics.groovy

```diff
@@ -37,6 +37,7 @@ class SnowflakeCollectMetrics {
                 environmentVariables {
                     env('SNOWFLAKE_USER', 'SNOWFLAKE_TASK_AUTOMATION_USER')
                     env('SNOWFLAKE_ACCOUNT', 'edx.us-east-1')
+                    env('SNOWFLAKE_WAREHOUSE', 'LOADING')
```

Comment:
I feel like this will make the job of tracking cost/usage confusing. Should we create or use another warehouse here?

Reply:
At some point yes, I will backlog a ticket to do so. For now I don't want to keep spinning up more warehouses that aren't doing much. For now we consolidate and cut costs. Once we have more things happening for monitoring we'll add a monitoring specific warehouse.